iml-api/src/timer.rs

```diff
@@ -0,0 +1,67 @@
+use crate::error::ImlApiError;
+use iml_manager_client::{get_client, put};
+use std::time::Duration;
+
+#[derive(serde::Serialize)]
+pub struct TimerConfig {
+    config_id: i32,
+    timer_config: String,
+    service_config: String,
+}
+
+pub async fn configure_snapshot_timer(
+    config_id: i32,
+    fsname: String,
+    interval: Duration,
+) -> Result<(), ImlApiError> {
+    let iml_cmd = format!(
+        r#"date +"%Y-%m-%dT%T%z" | xargs -I % echo "iml snapshot create {} {}-{}-%"#,
```

Comment:
This should always contain a comment like:

`automatically created by IML`

Reply:
Isn't that comment inside both the timer and the service?

Reply:
No, it's an actual param that needs to be passed to this cli.

Reply:
I was thinking the comment in the file. I understand, I'll add it with the -c option.